### DISCUSSION TITLE
Unable to make it to the setup screen (Ventura, Monterey & Big Sur)

### DISCUSSION THREAD

**Discussion by King91OM** [notAperson535/OneClick-macOS-Simple-KVM#78] [Q&A] [answered]
Hi all,

Hoping to get some help. Followed by the book with the guide & is unable to get the MacOS to launch (tried with Ventura, Monterey & Big Sur). It installs the Macos but it just won't launch it. When I select MacOS base system, it will loop back to the MacOS recovery. I've tried reinstalling many times with different MacOS to no avail. I am using Dell Precision 5530, Intel Xeon E3-1505M v6. 
Yes, I installed using macOS Extended Journaled non case-sensitive.

I have included my basic.sh & the output text when I run the MacOS. (I have tried using -cpu host but it still doesn't work)

![Screenshot 2023-04-27 154348](https://user-images.githubusercontent.com/10276539/234797624-1e481dc0-caa7-4ab9-ba79-c709131ae613.png)
![Screenshot 2023-04-27 155320](https://user-images.githubusercontent.com/10276539/234797667-42bf7666-57d2-432c-a37f-9a66bc271b14.png)
[basic.txt](https://github.com/notAperson535/OneClick-macOS-Simple-KVM/files/11341027/basic.txt)
[Ubuntu output file.txt](https://github.com/notAperson535/OneClick-macOS-Simple-KVM/files/11341031/Ubuntu.output.file.txt)

**Comment by Thultanthar** [comment, marked as answer]
You’ll need to select “macOS Installer” to continue the installation. Each reboot will require you to do this from my experience during install.

**Reply by King91OM** [reply]
You are absolutely right. Literally just press 4 times on the macOS installer & it finally reaches the setup screen. Thanks for the tip. Can finally proceed with the next phase :)

**Reply by notAperson535** [reply]
Glad you got that fixed!
